### PR TITLE
Cleaned up code as per pull request comments

### PR DESCRIPTION
**Pull request by athwds**

### CODE REVIEW

File: digikam_db.py [hunk 1]
```diff
@@ -50,14 +50,22 @@ def volume_uuid_to_mountpoints(uuid: str) -> Set[str]:
...
+                if specific_path.startswith('/'):
+                    specific_path = specific_path[1:]
+                for mountpoint in volume_uuid_to_mountpoints(identifier[15:]):
+                    self.album_roots[Path(mountpoint) / specific_path] = id
+            elif identifier.startswith('volumeid:?path='):
+                self.album_roots[identifier[15:]] = id
+            elif identifier.startswith('networkshareid:?mountpath='):
+                self.album_roots[identifier[26:]] = id
+            else:
+                raise ValueError('Unsupported ...' % identifier)
```
Comment (Philipp91) [inline digikam_db.py]: This isn't valid Python, sorry, my comment was abbreviated. The full line should be sth like:

```
raise ValueError('Unsupported volume type %s' % identifier)
```

(See https://stackabuse.com/python-string-interpolation-with-the-percent-operator/)

**Review by Philipp91 — CHANGES_REQUESTED** [review]

**Comment by athwds** [comment]
Good eye catching the missing "%s"!

**Comment by Philipp91** [comment]
Thank you!

**Comment by athwds** [comment]
NP! Now I'm working on adding support for very old [Contacts] section used from before about 2007.

**Pull request merged**